netlib/http2/protocol.py

```diff
@@ -127,12 +136,16 @@ def _create_headers(self, headers, stream_id, end_stream=True):
 
         header_block_fragment = self.encoder.encode(headers)
 
-        bytes = frame.HeadersFrame(
+        frm = frame.HeadersFrame(
             state=self,
             flags=flags,
             stream_id=stream_id,
-            header_block_fragment=header_block_fragment).to_bytes()
-        return [bytes]
+            header_block_fragment=header_block_fragment)
+
+        if self.dump_frames:  # pragma no cover
+            print(frm.human_readable(">>"))
+
+        return [frm.to_bytes()]
```

Comment:
It might not only be a single frame we want to return here.
Once we support ContinuationFrames we want to return a list of frames. 
Therefore using an array here is future-proof.